### ISSUE TITLE
Cobra MkII Cockpit Hatch

### ISSUE THREAD

**Issue by TheKurgan** [WolfairCorp/AoA-Technologies#11]
Hello, 
I noticed this was mentioned on the forum thread, and I thought I would open an issue here.

Love this mod, have used it since it came out long ago.

Any luck getting the unity files from Martinez?

**Comment by WolfairCorp** [comment]
Yeah I got them, but I had exams, so I was pretty disconected from KSP entirely. I'll see what I can do during christmas break.
Also sorry for late response.

**Comment by TheKurgan** [comment]
Awesome! Thanks, and Merry Christmas ;)

**Comment by WolfairCorp** [comment]
You too!
